src/components/Toggle/index.tsx

```diff
@@ -76,7 +76,7 @@ const styles = StyleSheet.create({
   },
   offText: {
     marginHorizontal: 3.5,
-    color: COLORS.lightgrey,
+    color: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is also in line with the PR's objective. Make sure `COLORS.gray100` is defined and matches the intended color code.

```diff
-    color: COLORS.lightgrey,
+    color: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->